### PR TITLE
Feature/34 admin invoice show item info

### LINKED ISSUE #40: feature/34-admin-invoice-show-item-info
34. Admin Invoice Show Page: Invoice Item Information

As an admin
When I visit an admin invoice show page (/admin/invoices/:invoice_id)
Then I see all of the items on the invoice including:
- Item name
- The quantity of the item ordered
- The price the Item sold for
- The Invoice Item status

### PR DESCRIPTION
**Pull request by AlexKiselich**
## Checklist before requesting a review
- [x] Verified Render deploy is functional
- [x] Pulled/merged main 
- [x] Tests for the changes have been added into spec files
- [x] Branch name follows our guidelines 
- [x] All tests running clean
- [x] Ran this in Rails S  - features are working properly
- [x] Test coverage is at 100% for models and features

## What changes or new behavior does this PR introduce (Bug fix, feature, update, etc)?
  Added 3 new instance methods to Items model (and added 2 new relationships) 
  Added tests for new instance methods and relationships
  Updated Admin/Invoice show page to include all items on the invoice
  Added test for admin/show spec
  Completed US34
## Bugs, Issues, etc:
 - [ ] check if there are known bugs or issue

### CODE REVIEW

File: spec/features/admin/invoices/show_spec.rb [hunk 1]
```diff
@@ -1,37 +1,63 @@
...
+        "Customer Name: #{@invoice_1.customer.first_name} #{@invoice_1.customer.last_name}"
+)
+    end
+
+    it "displays all the Items on that invoice and all the items information" do
+      visit admin_invoice_path(@invoice_1)
+      
+      within("#item_#{@item_1.id}") do
+        expect(page).to have_content("Item name: #{@item_1.name}")
+        expect(page).to have_content("Item Sold Price: $#{@invoice_item_1.unit_price/ 100.to_f}")
```
Comment (aplasket) [inline spec/features/admin/invoices/show_spec.rb:52]: hey @AlexKiselich i think you may be missing a test for quantity ordered. I see it in the view but not in these within blocks. is it somewhere else?

**Review by aplasket — CHANGES_REQUESTED** [review]
just missing the assertion in the within blocks about the quantity items (which you've tested for in the model, but likely should also test for in the spec for the view)

File: spec/features/admin/invoices/show_spec.rb (hunk 1 above)
Comment (AlexKiselich) [reply spec/features/admin/invoices/show_spec.rb:52]: Great catch! Updated

**Review by aplasket — APPROVED** [review]

File: app/models/item.rb
Comment (kbergstrom78) [inline app/models/item.rb:1]: That's some sweet, sweet AR right thar.

**Review by kbergstrom78 — APPROVED** [review]
Nice job, yet again!

**Pull request merged**